web/template/create-draft.gohtml

```diff
@@ -244,7 +244,7 @@
           </div>
 
           <div class="govuk-button-group">
-            <button class="govuk-button" data-module="govuk-button" type="submit">Save draft LPA</button>
+            <input type="submit" class="govuk-button" value="Save draft LPA" data-disable-after-click="true">
```

Comment:
I generally prefer `<button>` for buttons because it's more semantic and more flexible with content/styling.

Reply:
I generally prefer inputs with type submit :) But that's because I'm old fashioned. I'll try changing it back to a button, though this again seems to cause problems with the event handlers.